src/structured-data/handler.js

```diff
@@ -28,6 +28,9 @@ import { generatePlainHtml, getScrapeForPath } from '../support/utils.js';
 const auditType = Audit.AUDIT_TYPES.STRUCTURED_DATA;
 const auditAutoSuggestType = Audit.AUDIT_TYPES.STRUCTURED_DATA_AUTO_SUGGEST;
 
+// Limit to avoid excessive Firefall requests. Can be increased if needed.
+const FIREFALL_REQUEST_LIMIT = 50;
```

Comment:
should this be made configurable via AWS secrets in case we need to play?

Reply:
Thanks for the suggestion. I added a new `AUDIT_STRUCTURED_DATA_FIREFALL_REQ_LIMIT` secret to the prod and ci secret manager.